init.sh

```diff
@@ -41,7 +41,7 @@ select example in "1" "2" "3"; do
     case $example in
         1 ) 
             myhost=`ifconfig eth1 | grep inet | cut -d':' -f2 | cut -d' ' -f1 | sed 's/\./\-/g'`
-            export ENDPOINT_HOST="ip${myhost}-${SESSION_ID}-5001.direct.labs.play-with-docker.com"
+            export ENDPOINT_HOST="ip${myhost}-${SESSION_ID}-5001.direct.play-with-docker.vonx.io"
```

Comment:
There is an environment variable - PWD_HOST_FQDN=play-with-docker.vonx.io  that we can use for the last part of this.  

Change this to:

`export ENDPOINT_HOST="ip${myhost}-${SESSION_ID}-5001.direct.${PWD_HOST_FQDN}"`